tests/system_tests/can/python_can/test_vector.py

```
@@ -0,0 +1,66 @@
+    AbstractUnsegmentedMessageTests,
+    AbstractUseCaseTests,
+)
+
+# Config
+
+class VectorConfig(AbstractPythonCanTests):
+    """Configuration for python-can Transport Interface tests with Vector CAN interfaces."""
+
+    MAKE_TIMING_CHECKS: bool = True
+
+    def _define_interfaces(self):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_define_interfaces`

Add return type annotation: `None`

(ANN202)